images/router/f5/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#
+# This is the HAProxy router for OpenShift Origin.
+#
+# The standard name for this image is openshift/origin-f5-router
+#
+FROM openshift/origin-base
+
+ADD bin/openshift /usr/bin/openshift
+RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
+
+ENTRYPOINT ["/usr/bin/openshift-router", "--type=f5"]
```

Comment:
I see what you mean.  An alternative would be to make `template` the default value for `--type`, but it would be cleaner to modify the haproxy router's `Dockerfile`, so I'll do that.  Good catch.